src/Godot.SourceGenerators/BindMethodsWriter.cs

```diff
@@ -140,7 +140,12 @@ private static void WriteCachedStringNames(IndentedStringBuilder sb, GodotClassS
 
         void AddCachedStringName(string symbolName, string value)
         {
-            sb.AppendLine($$"""public static global::Godot.StringName @{{symbolName}} { get; } = global::Godot.StringName.CreateStaticFromAscii("{{value}}"u8);""");
+            if (value.IsAscii()) {
+                sb.AppendLine($$"""public static global::Godot.StringName @{{symbolName}} { get; } = global::Godot.StringName.CreateStaticFromAscii("{{value}}"u8);""");
+            } else {
+                sb.AppendLine($$"""public static global::Godot.StringName @{{symbolName}} { get; } = global::Godot.StringName.CreateFromUtf8("{{value}}"u8);""");
```

Comment:
To match our code-style, we use the "Allman" style which also matches [C# style guidelines](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#style-guidelines).

```suggestion
            if (value.IsAscii())
            {
                sb.AppendLine($$"""public static global::Godot.StringName @{{symbolName}} { get; } = global::Godot.StringName.CreateStaticFromAscii("{{value}}"u8);""");
            }
            else
            {
                sb.AppendLine($$"""public static global::Godot.StringName @{{symbolName}} { get; } = global::Godot.StringName.CreateFromUtf8("{{value}}"u8);""");
```